### ISSUE TITLE
NTR: Mycobacterium present in CSF culture (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8059]
New term request Mycobacterium present in CSF culture (samant_101122005934): Cerebrospinal fluid  UMLS: 2026564

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000543
- label: Posiitive mycobacterium CSF culture
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: 
- def: Growth of mycobacterium in a culture derived from cerebrospinal fluid (CSF).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8059
- synonyms:
